opentracing-metrics-prometheus-spring-autoconfigure/src/test/java/io/opentracing/contrib/metrics/prometheus/spring/autoconfigure/PrometheusMetricsReporterConfigurationTest.java

```diff
@@ -49,24 +47,47 @@
     public static class SpringConfiguration {
         @Bean
         public CollectorRegistry testCollectorRegistry() {
+            System.out.println("Producing registry " + testCollectorRegistry);
```

Comment:
Did you want this diagnostics?

Reply:
If so, might want to put in function -as same code used below.

Reply:
I did but will remove once the situation I described in the first comment is sorted out.